examples/02-Maxwell/Maxwell3DTeam7.py

```diff
@@ -0,0 +1,380 @@
+
+# Add a new material with Team 7 properties for Aluminium
+mat = M3D.materials.add_material("team7_aluminium")
+mat.update()
+mat.conductivity = 3.526e7
+
+# Model the Aluminium plate with a hole in by subtracting two rectangular cuboids.
+plate = M3D.modeler.primitives.create_box([0, 0, 0], [294, 294, 19], name="Plate", matname="team7_aluminium")
+M3D.modeler.fit_all()
+hole = M3D.modeler.primitives.create_box([18, 18, 0], [108, 108, 19], name="Hole")
+M3D.modeler.subtract("Plate", ["Hole"], keepOriginals=False)
+
+# Draw a background region, it will take default Air properties
+M3D.modeler.create_air_region(x_pos=100, y_pos=100, z_pos=100, x_neg=100, y_neg=100, z_neg=100)
+
+################################################################################
+# Adjust Eddy Effects for Plate and Coil
+# Turn off displacement currents for all parts
+# NB, Eddy effect setting is regardless ignored for stranded conductor type used in Coil
+
+test = M3D.odesign.GetModule("BoundarySetup")
+test.SetEddyEffect(
+    [
+        "NAME:Eddy Effect Setting",
+        [
+            "NAME:EddyEffectVector",
+            ["NAME:Data", "Object Name:=", "Plate", "Eddy Effect:=", True, "Displacement Current:=", False],
+            ["NAME:Data", "Object Name:=", "Coil", "Eddy Effect:=", False, "Displacement Current:=", False],
+            ["NAME:Data", "Object Name:=", "Region", "Eddy Effect:=", False, "Displacement Current:=", False],
+            ["NAME:Data", "Object Name:=", "Line_A1_B1mesh", "Eddy Effect:=", False, "Displacement Current:=", False],
+            ["NAME:Data", "Object Name:=", "Line_A2_B2mesh", "Eddy Effect:=", False, "Displacement Current:=", False],
+        ],
+    ]
+)
+
+################################################################################
+# Use Fields Calculator to create an expression for Z Component of B in Gauss
+Fields = M3D.odesign.GetModule("FieldsReporter")
+Fields.EnterQty("B")
+Fields.CalcOp("ScalarZ")
+Fields.EnterScalarFunc("Phase")
+Fields.CalcOp("AtPhase")
+Fields.EnterScalar(10000)
+Fields.CalcOp("*")
+Fields.CalcOp("Smooth")
+Fields.AddNamedExpression("Bz", "Fields")
+
+################################################################################
+# Draw two lines along which we will plot Bz
+# A small cylinder is also added to refine mesh locally around each line.
+Lines = ["Line_A1_B1", "Line_A2_B2"]
+MeshDia = "2mm"
+
+Line_Points1 = [["0mm", "72mm", "34mm"], ["288mm", "72mm", "34mm"]]
+L1 = primitives.create_polyline(Line_Points1, name=Lines[0])
+L1Mesh = primitives.create_polyline(Line_Points1, name=Lines[0] + "mesh")
+L1Mesh.set_crosssection_properties(type="Circle", width=MeshDia)
+
+Line_Points1 = [["0mm", "144mm", "34mm"], ["288mm", "144mm", "34mm"]]
+L2 = primitives.create_polyline(Line_Points1, name=Lines[1])
+L2Mesh = primitives.create_polyline(Line_Points1, name=Lines[1] + "mesh")
+L2Mesh.set_crosssection_properties(type="Circle", width=MeshDia)
```

Comment:
If possible, please follow snake_case variable naming convention. 
```suggestion
lines = ["Line_A1_B1", "Line_A2_B2"]
mesh_diameter = "2mm"

line_points_1 = [["0mm", "72mm", "34mm"], ["288mm", "72mm", "34mm"]]
polyline = primitives.create_polyline(line_points_1, name=Lines[0])
L1Mesh = primitives.create_polyline(line_points_1, name=Lines[0] + "mesh")
L1Mesh.set_crosssection_properties(type="Circle", width=mesh_diameter)

line_points_2 = [["0mm", "144mm", "34mm"], ["288mm", "144mm", "34mm"]]
polyline2 = primitives.create_polyline(line_points_2, name=Lines[1])
polyline2_mesh= primitives.create_polyline(line_points_2, name=Lines[1] + "mesh")
polyline2_mesh.set_crosssection_properties(type="Circle", width=mesh_diameter)
```

Reply:
Good point thanks for adjusting